docs/release/release-tasks.md

```diff
@@ -335,7 +337,8 @@ The goal of this task to make the book for the current release available under `
 
 Someone with access to Netlify should:
 1. Change production branch in Netlify the current release branch (e.g. `release-1.4`) to make the book available under `https://cluster-api.sigs.k8s.io`.
-2. Re-deploy via the Netlify UI.
+2. Re-deploy via the Netlify [production branch settings](https://app.netlify.com/sites/kubernetes-sigs-cluster-api/settings/deploys#branches-and-deploy-contexts).
```

Comment:
Not sure this is pointing to the right URL, the "trigger button" is how I've done this in the past on this page https://app.netlify.com/sites/kubernetes-sigs-cluster-api/deploys

Reply:
I think you're right. My intention was to suggest this URL for the step above to change the production branch setting

Reply:
Let's use this URL for the step above and your URL + "Trigger deploy" button for this step